grammars/ruby.cson

```diff
@@ -31,7 +31,7 @@
   'Thorfile'
   'Vagrantfile'
 ]
-'firstLineMatch': '^#!\\s*/.*\\bruby'
+'firstLineMatch': '^#!\\s*/.*\\bruby|^#\\s+-\\*-\\s*ruby\\s*-\\*-",
```

Comment:
This needs to end in a `'` instead of a `"`, and the `,` isn't needed after the quote